src/TimeSeries.jl

```diff
@@ -19,12 +19,12 @@ export TimeArray, AbstractTimeSeries,
 
 include(".timeseriesrc.jl")
 include("timearray.jl")
+include("utilities.jl")
 include("split.jl")
 include("apply.jl")
 include("broadcast.jl")
 include("combine.jl")
 include("readwrite.jl")
-include("utilities.jl")
 include("modify.jl")
 include("basemisc.jl")
 include("deprecated.jl")
```

Comment:
why keeping and empty `deprecated.jl` file.

https://github.com/JuliaStats/TimeSeries.jl/blob/4fbedc3f4c162e09600d19d5fb2d633f3c8a1aee/src/deprecated.jl

Reply:
I want to keep `src/deprecated.jl` for future use, but I do not want to test the deprecation binding in the future, since it mess up the testing output. I just removed the `test/deprecated.jl`